README.md

```diff
@@ -151,6 +151,25 @@ bash build-and-publish-docker-image.sh --image <repo>/<imagename>:<imagetag>
 ```
 > Note: format of the imagetag will be `ci<release><MMDDYYYY>`. possible values for release are test, dev, preview, dogfood, prod etc.
 
+You can also build and push images for multiple architectures. This is powered by docker buildx
+```
+cd ~/Docker-Provider/kubernetes/linux/dockerbuild
+sudo docker login # if you want to publish the image to acr then login to acr via `docker login <acr-name>`
+# build and publish using docker buildx
+bash build-and-publish-docker-image.sh --image <repo>/<imagename>:<imagetag> --multiarch
```

Comment:
do we need this --multiarch? cant we make without this flag to build multiarch image?

Reply:
so the reason I added this --multiarch option is to keep backwards compatibility. if someone wants to use the old way they can just call the existing command without --multiarch